detox/test/package.json

```diff
@@ -43,6 +43,7 @@
     "detox": "^19.1.0",
     "dtslint": "^4.0.6",
     "express": "^4.15.3",
+    "glob": "^7.2.0",
```

Comment:
Is it really necessary? if we only need to filter files by their extension, we already have [path.extname()](https://nodejs.org/api/path.html#pathextnamepath) method for this

Reply:
I use mask to assert in E2E tests:

https://github.com/wix/Detox/pull/3102/files#diff-74e2304051bae3742d988989259ca506350833a28a82d991fce51c513f7d6f05R23

and by the way it is already used by dependencies of dependencies, and it is just e2e self-test project, so... 

WDYT?

Reply:
OK, keep it then 👍🏼